### PR TITLE
Site overhaul

### PR DESCRIPTION
**Pull request by 404invalid-user**

### CODE REVIEW

**Comment by cobaltgit** [comment]
Damn I can't merge it
image

File: good-styles.css
Comment (sabianroberts) [inline good-styles.css]: You need to add a webfont kit for Helvetica since only Macs come with the font already installed.

File: index.html
Comment (sabianroberts) [inline index.html:15]: The file "olddigibox.png" isn't in your fork; did you forget to add it?

**Review by sabianroberts — CHANGES_REQUESTED** [review]
Helvetica webfont kit needs adding and olddigibox.png (see comments on files)

**Comment by sabianroberts** [comment]
> Damn I can't merge it
> image

I've requested some changes; you need to read the code carefully rather than looking at it once and going "oh that looks nice, I'll merge"

There's just a missing webfont kit and an image I think

File: good-styles.css
Comment (404invalid-user) [reply good-styles.css]: I use a mac I didn't know that. I have removed that font and added another

File: index.html
Comment (404invalid-user) [reply index.html:15]: Yes it is there now

**Comment by 404invalid-user** [comment]
Should be done now

**Review by sabianroberts — APPROVED** [review]

**Pull request merged**
